src/Kunstmaan/UtilitiesBundle/Helper/Shell/Shell.php

```diff
@@ -2,39 +2,48 @@
 
 namespace Kunstmaan\UtilitiesBundle\Helper\Shell;
 
+use Symfony\Component\Process\Process;
+
 /**
  * A wrapper class which makes it possible to execute shell commands in the background.
  */
 class Shell implements ShellInterface
 {
 
     /**
-     * @param string $command  The command
-     * @param int    $priority The priority
+     * @param string $command The command
      *
-     * @return string The process id
+     * @return int The process id
+     * @throws \Symfony\Component\Process\Exception\LogicException
+     * @throws \Symfony\Component\Process\Exception\RuntimeException
      */
-    public function runInBackground($command, $priority = 0)
+    public function runInBackground($command)
```

Comment:
@cv65kr 
Ok you removed the argument to it completely now. This would make it a BC break and would need to go in 5.0 branch.

Please choose one of both options.

- rebase branch to 5.0
OR
- add the argument back in the function definiton but not use it in the implementation and add Deprecated annotation to that argument.

Reply:
@Numkil done, if you merge i will sent next PR for 5.0 version.